feg/protos/s6a_proxy.proto

```diff
@@ -52,14 +52,14 @@ enum ErrorCode {
     MULTI_ROUND_AUTH         = 1001;
     SUCCESS                  = 2001;
     LIMITED_SUCCESS          = 2002;
-    COMMAND_UNSUPORTED       = 3001;
+    COMMAND_UNSUPPORTED      = 3001;
     UNABLE_TO_DELIVER        = 3002;
     REALM_NOT_SERVED         = 3003;
     TOO_BUSY                 = 3004;
     LOOP_DETECTED            = 3005;
     REDIRECT_INDICATION      = 3006;
     APPLICATION_UNSUPPORTED  = 3007;
-    INVALIDH_DR_BITS         = 3008;
+    INVALID_HDR_BITS         = 3008;
```

Comment:
@ssanadhya, can we modify the proto field as it is not backward compatible. But error codes have fixed values.

Reply:
I think it's no problem for binary serialization (see for example https://stackoverflow.com/a/50863982). I am not 100% sure that's what we use in this interface though.

Reply:
As per my understanding field and values of proto needs to be same across various release versions.

Reply:
@rsarwad , this should be okay, since it is not changing an actual field in the proto message, but the name of an Enum value. As @sebathomas highlighted, the value 3008 will be sent on the wire with type as int. 

Further, since these are standard Diameter error codes, we would want to map 3008 to "Invalid header bits"

Reply:
Thanks @ssanadhya  for clarification